.github/workflows/auto-respond-pr.yml

```diff
@@ -14,14 +14,15 @@
 name: Auto Respond to PR
 
 on:
-    pull_request:
+    pull_request_review:
```

Comment:
This is currently triggered on all activity types for `pull_request_review` (submitted, edited or dismissed). It could be restricted to e.g. just `submitted` but if you’re happy with it as-is, that’s fine.

Reply:
These are required for the asana sync further down. We may wish to cache or avoid some steps but other than that it’s intended so we sync properly.